                                                                                                                                                                                                                                                               
Delivered-To: jwboyer@gmail.com
Received: by 10.76.6.212 with SMTP id d20csp275723oaa;
        Wed, 11 Jun 2014 09:20:26 -0700 (PDT)
X-Received: by 10.68.202.194 with SMTP id kk2mr6443717pbc.156.1402503625476;
        Wed, 11 Jun 2014 09:20:25 -0700 (PDT)
Return-Path: <luto@amacapital.net>
Received: from bastion.fedoraproject.org (bastion02.fedoraproject.org. [209.132.181.3])
        by mx.google.com with ESMTP id hk10si7140166pac.55.2014.06.11.09.20.25
        for <jwboyer@gmail.com>;
        Wed, 11 Jun 2014 09:20:25 -0700 (PDT)
Received-SPF: softfail (google.com: domain of transitioning luto@amacapital.net does not designate 209.132.181.3 as permitted sender) client-ip=209.132.181.3;
Authentication-Results: mx.google.com;
       spf=softfail (google.com: domain of transitioning luto@amacapital.net does not designate 209.132.181.3 as permitted sender) smtp.mail=luto@amacapital.net
Received: by bastion02.phx2.fedoraproject.org (Postfix)
	id F167F40A24; Wed, 11 Jun 2014 16:20:24 +0000 (UTC)
Delivered-To: jwboyer@fedoraproject.org
Received: from mx1.redhat.com (ext-mx12.extmail.prod.ext.phx2.redhat.com [10.5.110.17])
	by bastion02.phx2.fedoraproject.org (Postfix) with ESMTP id E054140A1F
	for <jwboyer@fedoraproject.org>; Wed, 11 Jun 2014 16:20:24 +0000 (UTC)
Received: from mail-pd0-f174.google.com (mail-pd0-f174.google.com [209.85.192.174])
	by mx1.redhat.com (8.14.4/8.14.4) with ESMTP id s5BGKK9D017943
	for <jwboyer@fedoraproject.org>; Wed, 11 Jun 2014 12:20:21 -0400
Received: by mail-pd0-f174.google.com with SMTP id y10so1503744pdj.19
        for <jwboyer@fedoraproject.org>; Wed, 11 Jun 2014 09:20:20 -0700 (PDT)
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
        d=1e100.net; s=20130820;
        h=x-gm-message-state:from:to:cc:subject:date:message-id:in-reply-to
         :references:in-reply-to:references;
        bh=CQ9F+PXV6SvMruBV8gwiVUY2iUpI65+jqkUh4N/yL04=;
        b=PJ/CK5msKabsmSWS6VSEgjtZ7jgX0BF/9CHTigKuAr0W/4IU0xN0osE/pc6Gny5slE
         /0PZrvgDnSLczb+l7Ed2kovova/KZktb6EbUXeOJEYfKC9dhytsCYDG9tuLUvxEmgZfp
         vOum3H1CA3ocdu+5qKvwem2WOdLu2+2t3i8KchcmsDZcWhK66eb8nro1LbbwWgdKsPSF
         V5NT773uUJ23hy2/uGO6m/PiSIrc8skrgCEae6ugoL0M5r+H9UwfqvLlYB1PL+DJ2JPm
         t4wLxV2XxD610nR3sXIs1urvX+/4xAG+2ZdC8NMi7+A2SVtku64rS/5UI30D00xfQoB6
         hYLg==
X-Gm-Message-State: ALoCoQnsNny2k21X9YSmrkqYkA6qcZffpVvIHQPZiz91j0A2KaYHwDQ4WNaxXqR1ZXSsLA8qPd+g
X-Received: by 10.66.172.229 with SMTP id bf5mr14570106pac.57.1402503620494;
        Wed, 11 Jun 2014 09:20:20 -0700 (PDT)
Received: from localhost (50-76-60-73-ip-static.hfc.comcastbusiness.net. [50.76.60.73])
        by mx.google.com with ESMTPSA id fk4sm31444487pab.23.2014.06.11.09.20.19
        for <multiple recipients>
        (version=TLSv1.2 cipher=ECDHE-RSA-AES128-GCM-SHA256 bits=128/128);
        Wed, 11 Jun 2014 09:20:19 -0700 (PDT)
From: Andy Lutomirski <luto@amacapital.net>
To: "H. Peter Anvin" <hpa@zytor.com>, Josh Boyer <jwboyer@fedoraproject.org>
Cc: Michal Marek <mmarek@suse.cz>, linux-kbuild@vger.kernel.org,
        "Linux-Kernel@Vger. Kernel. Org" <linux-kernel@vger.kernel.org>,
        Andy Lutomirski <luto@amacapital.net>
Subject: [PATCH 1/2] kbuild: Add recipe-cmd, an @-less cmd variant
Date: Wed, 11 Jun 2014 09:20:06 -0700
Message-Id: <b3ea9607ad67524018c336e1dd54f5d729d60324.1402503408.git.luto@amacapital.net>
X-Mailer: git-send-email 1.9.3
In-Reply-To: <cover.1402503408.git.luto@amacapital.net>
References: <5398749B.4090209@zytor.com>
 <cover.1402503408.git.luto@amacapital.net>
In-Reply-To: <cover.1402503408.git.luto@amacapital.net>
References: <cover.1402503408.git.luto@amacapital.net>
X-RedHat-Spam-Score: -2.01  (BAYES_00,DCC_REPUT_13_19,RCVD_IN_DNSWL_NONE,SPF_PASS,URIBL_BLOCKED)
X-Scanned-By: MIMEDefang 2.68 on 10.5.110.17

It's awkward to put $(call cmd,xyz) in a loop in a recipe because
the pesky at sign gets in the way.

Signed-off-by: Andy Lutomirski <luto@amacapital.net>
---
 scripts/Kbuild.include | 1 +
 1 file changed, 1 insertion(+)

diff --git a/scripts/Kbuild.include b/scripts/Kbuild.include
index 122f95c..3e60923 100644
--- a/scripts/Kbuild.include
+++ b/scripts/Kbuild.include
@@ -194,6 +194,7 @@ echo-cmd = $(if $($(quiet)cmd_$(1)),\
 
 # printing commands
 cmd = @$(echo-cmd) $(cmd_$(1))
+recipe-cmd = $(echo-cmd) $(cmd_$(1))
 
 # Add $(obj)/ for paths that are not absolute
 objectify = $(foreach o,$(1),$(if $(filter /%,$(o)),$(o),$(obj)/$(o)))
-- 
1.9.3

